gitprompt.sh

```diff
@@ -603,6 +603,10 @@ function updatePrompt() {
 # Helper function that returns virtual env information to be set in prompt
 # Honors virtualenvs own setting VIRTUAL_ENV_DISABLE_PROMPT
 function gp_add_virtualenv_to_prompt {
+  # Exit early if prompt should not include virtual env info
+  if [[ "${GIT_PROMPT_WITH_VIRTUAL_ENV:-1}" == 0 ]]; then
+    return
+  fi
```

Comment:
Please move the check out of here and do it like it is done in the `git_prompt_config` by checking if `GIT_PROMPT_WITH_VIRTUAL_ENV` is set and if so, build the `NEW_PROMPT` var with respect to `gp_add_virtualenv_to_prompt` and if the var is not set, build `NEW_PROMPT` without the use of `gp_add_virtualenv_to_prompt`